### PR TITLE
fix #54041 t.myvisualiq.net

### LINKED ISSUE #54041: t.myvisualiq.net
### Issue URL (Anti Adblock Script)
 [https://t.myvisualiq.net/click_pixel?et=c&ago=212&ao=923&aca=-4&si=398083&ci=0&pi=fb-social-da~2004182351~2004201101~20110631&ad=-4&sv1=-4&advt=-4&chnl=-4&vndr=1268&sz=9400&u=-4&red=https://www.thalia.de/shop/home/artikeldetails/ID147587094.html?pid=398083-fb-social-da~2004182351~2004201101~20110631&partner_ref=fb-social-da~2004182351~2004201101~20110631&contactdate=1587373594&group_id=0&banner_id=0&product_id=0&utm_medium=affiliate&utm_source=www.brickmerge.de&awc=14158_1587373594_d8d95d96cc456c418f6d620a8a1a7824&ProvID=10911403](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ft.myvisualiq.net%2Fclick_pixel%3Fet%3Dc%26ago%3D212%26ao%3D923%26aca%3D-4%26si%3D398083%26ci%3D0%26pi%3Dfb-social-da~2004182351~2004201101~20110631%26ad%3D-4%26sv1%3D-4%26advt%3D-4%26chnl%3D-4%26vndr%3D1268%26sz%3D9400%26u%3D-4%26red%3Dhttps%3A%2F%2Fwww.thalia.de%2Fshop%2Fhome%2Fartikeldetails%2FID147587094.html%3Fpid%3D398083-fb-social-da~2004182351~2004201101~20110631%26partner_ref%3Dfb-social-da~2004182351~2004201101~20110631%26contactdate%3D1587373594%26group_id%3D0%26banner_id%3D0%26product_id%3D0%26utm_medium%3Daffiliate%26utm_source%3Dwww.brickmerge.de%26awc%3D14158_1587373594_d8d95d96cc456c418f6d620a8a1a7824%26ProvID%3D10911403)
### Comment
Username: @NeXrom
> Der Link wird wegen Affilate Link blockiert aufgrund des AdGuard Basis-filter. Wie kann ich hier eine Ausnahme erstellen? Habe bereits Die Website t.myvisualiq.net gewhitlistet. Funktioniert aber trotzdem nicht
> 
> The link is blocked because of the affiliate link due to the AdGuard basic filter. How can I create an exception here? Have already listed the website t.myvisualiq.net. But still doesn't work
> 
> Is it possible to make an exception to all affiliate Links? Cause I don't care who gets money :-P
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 1.7.4
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#54041
On brickmerge.de affiliate links to thalia.de is blocked in Safari because of `&banner_id=` rule from the base filter.

<a href="https://www.brickmerge.de/42083-1_lego-technic-bugatti-chiron">brickmerge.de example page</a>
<details>
<summary>Screenshot</summary>

</details>

### CODE REVIEW

File: EnglishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -214,6 +214,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55620
 @@||edge.microsoft.com/translate/auth
 @@||api.cognitive.microsofttranslator.com/translate?
+! https://github.com/AdguardTeam/AdguardFilters/issues/54041
+@@||*thalia.de*&utm_source=www.brickmerge.de
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist.txt:218]: I see three URLs with `&banner_id=`
```
https://t.myvisualiq.net/click_pixel?*&red=https://www.thalia.de/shop/*&banner_id=*
https://t.myvisualiq.net/ul_cb/click_pixel?*&red=https://www.thalia.de/shop/*&banner_id=*
https://thalia.de/shop/*.html?pid=*&group_id=0&banner_id=
```
What exactly must be excluded?

Comment (Alex-302) [reply EnglishFilter/sections/whitelist.txt:218]: @Sergey-Lyapin

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:218]: It seems that such a rule is enough
```
!+ PLATFORM(ios, ext_safari)
@@||*thalia.de/shop/*.html?pid=*&group_id=0&banner_id=
```

Comment (Alex-302) [reply EnglishFilter/sections/whitelist.txt:226]: Why not just `@@thalia.de/shop/...`?

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt]: Yes, it looks better

File: EnglishFilter/sections/whitelist.txt [hunk 2]
```diff
@@ -222,6 +222,8 @@ metacritic.com#@#.esite_list
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55620
 @@||edge.microsoft.com/translate/auth
 @@||api.cognitive.microsofttranslator.com/translate?
+! https://github.com/AdguardTeam/AdguardFilters/issues/54041
+@@thalia.de*&utm_source=www.brickmerge.de
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist.txt:226]: where is the hint?

File: EnglishFilter/sections/whitelist.txt [hunk 3]
```diff
@@ -222,6 +222,9 @@ metacritic.com#@#.esite_list
 ! https://github.com/AdguardTeam/AdguardFilters/issues/55620
 @@||edge.microsoft.com/translate/auth
 @@||api.cognitive.microsofttranslator.com/translate?
+! https://github.com/AdguardTeam/AdguardFilters/issues/54041
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist.txt:225]: why not to `content_blocker.txt`?

**Pull request closed without merging**